Assignment-2-User-Management.c

```diff
@@ -0,0 +1,175 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+//defining the struct to store the details of the user
+struct user
+{
+    int id;
+    char name[50];
+    int age;
+};
+
+void main()
+{
+
+    int ch;

```

Comment:
take care of naming conventions for variables, functions and apply code formatting as suggested in calculator PR.